### PR TITLE
data:Kaif987

### PR DESCRIPTION
**Pull request by Kaif987**

### CODE REVIEW

File: data/Kaif987.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Mohammad Kaif",
+  "type": "personal",
+  "bio": "A MERN stack developer who loves programming",
+  "links": [
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/mohammad-kaif-4a0ab2231/",
+      "icon": "FaLinkedinr"
```
Comment (khushalarora) [inline data/Kaif987.json:9]: The current icon seems to be invalid. You might want to consider replacing it with **'FaLinkedin'** instead.
You can also refer to [ICONS](https://linkfree.eddiehub.io/icons) , thanks :+1: 
```Suggestion
      "icon": "FaLinkedin"
```

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hey @Kaif987, thank you for contributing :tada:

**Comment by Kaif987** [comment]
Sorry It was a silly mistake from my side.

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks for your Contribution! 🎉 

<p align="center"><a href="https://linkfree.io/docs/how-to-guides/editing"></a></p>

**Pull request merged**